app/assets/javascripts/discourse/app/mixins/card-contents-base.js

```diff
@@ -85,15 +90,12 @@ export default Mixin.create({
     const id = this.elementId;
     const triggeringLinkClass = this.triggeringLinkClass;
     const clickOutsideEventName = `mousedown.outside-${id}`;
-    const clickDataExpand = `click.discourse-${id}`;
```

Comment:
`.discourse-user-card` and `.discourse-group-card` and no longer used that I can find... Removing the extra selectors that don't do anything.